Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -463,14 +473,16 @@ def _async_fetch_epidata(data_source: str,
                          issues: Union[date, tuple, list],
                          lag: int,
                          time_type: str = "day") -> Union[pd.DataFrame, None]:
-    """Fetch data from Epidata API asynchronously.
+    """Fetch data from Epidata API asynchronously. [DEPRECATED]
```

Comment:
```suggestion
    """Fetch data from Epidata API asynchronously [DEPRECATED].
```
this is a stupid rule being applied to a private function, and its beginning to seriously irk me...